installer/docker/README.md

```diff
@@ -1,5 +1,125 @@
 # HNN Docker Container
 
+### ** UPDATED Instructions: 11/29/2023 **
+
+### Windows Installation Process
+
+## Part 1 Install WSL
+1. Open the Command Prompt application. Can be found by searching `Command Prompt` in the Windows search bar.
+2. Paste the following command to install Windows Subsystem for Linux (WSL)
+```
+wsl --install
```

Comment:
Is there a way to manually specify WSL2?

I think this is installed by default now but we've run into a few folks that had WSL1 and couldn't install docker

Reply:
Just for clarification, are u saying folks who already had WSL1 before attempting this installation process were having errors installing docker? Like you said, `wsl --install` should by default install WSL2 so users who are following this tutorial, starting from scratch, should have WSL2 installed. 

I could add some additional instructions here on how to upgrade from WSL1 to WSL2 if that helps

Reply:
Perhaps we could just say something like 
```
Note: if WSL is already installed on your computer, please [ensure it is WSL2 ](https://learn.microsoft.com/en-us/windows/wsl/install#check-which-version-of-wsl-you-are-running). WSL1 is incompatible with Docker install.
```

Reply:
Sounds good, i'll add that in